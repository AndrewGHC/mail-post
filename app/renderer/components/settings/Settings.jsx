// @flow
import React, { Fragment } from 'react';

const Settings = () => (
  <Fragment>
    Settings
  </Fragment>
);

export default Settings;
