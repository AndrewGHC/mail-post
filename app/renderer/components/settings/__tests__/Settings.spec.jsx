import React from 'react';
import { shallow } from 'enzyme';

import Settings from '../Settings';

describe('Settings', () => {
  it('renders elements', async () => {
    const wrapper = shallow(<Settings />);
    expect(wrapper).toMatchSnapshot();
  });
});
