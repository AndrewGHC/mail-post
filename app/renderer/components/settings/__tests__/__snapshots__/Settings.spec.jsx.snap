// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings matches snapshot 1`] = `
<React.Fragment>
  Settings
</React.Fragment>
`;
